docs/lab/overview/introduction.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: "Introduction"
+---
+
+Vital's lab test API allows digital health companies to carry out at-home lab testing.
+Companies can use the API to order a variety of tests, using various sampling methods.
+
+We partner with CLIA and CAP certified labs across the United States that cover a variety of diagnostic tests. For example lipids panels, metabolic profiles, hormone tests and more.
```

Comment:
Same situation with using `variety` again. You could use a `range of different diagnostic tests`